src/utils/is-meta-mask.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+function isMetaMask() {
```

Comment:
Use the same function here? https://github.com/ethereumjs/ethrpc/blob/7f63d147bdec1e17bab7618d28ed22f464c5ee6c/src/transport/web3-transport.js#L15

Reply:
It's not quite the same function, but yes there's some duplicate code, added a global web3 check function.